packages/flutter_blue/example/tizen/tizen-manifest.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="org.tizen.flutter_blue_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="common"/>
+    <service-application appid="org.tizen.flutter_blue_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" nodisplay="true" taskmanage="false" api-version="4" auto-restart="false">
+        <label>example</label>
+        <icon>ic_launcher.png</icon>
+        <background-category value="media"/>
+        <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true"/>
+    </service-application>
+    <tv-info api-version="8.8.0">
+        <infolink>T-INFOLINK2021-1000</infolink>
+    </tv-info>
+    <feature name="http://tizen.org/feature/screen.size.all"/>
+</manifest>
```

Comment:
Missing privilege, is it not required?

```xml
<privileges>
    <privilege>http://tizen.org/privilege/bluetooth</privilege>
</privileges>
```

I haven't read the implementation code yet so I don't know how the app is supposed to work, but isn't this a service application? The original capp was ui application: https://github.com/flutter-tizen/plugins/pull/355/commits/da57e6565754c2d41cda11e868aac1bd7471a6e5#diff-5eb7a9e892b5e46abd1d4213e963e37b8c6b4774fa74f82617804e163289a4f2.

Reply:
I made a mistake here. It should have privileges and be an `ui-application`